test/json_unittest.cc

```diff
@@ -0,0 +1,385 @@
+#include <functional>
+
+#include "../src/json.h"
+#include "gtest/gtest.h"
+
+#define EXPECT_TO_STRING(v, s) EXPECT_EQ(v->ToString(), s)
+
+namespace {
+
+void GuardJsonException(std::function<void()> test) {
+  try {
+    test();
+  } catch (const json::Exception& exc) {
+    FAIL() << exc.what();
+  }
+}
+
+
+// Trival Null
+
+TEST(TrivialParseTest, Null) {
```

Comment:
All of the Parse/ToString tests seem to have the "Trivial" prefix. Do we need it?

Reply:
I think it helps to easily identify the trivial ones from a set of failling tests, because fixing those first is usually easier and can sometimes implicitly fix all the others.